mgmt_upgrade_test.py

```diff
@@ -118,23 +110,24 @@ def test_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
         validate_previous_task_details(task=repair_task, previous_task_details=repair_task_current_details)
         validate_previous_task_details(task=backup_task, previous_task_details=backup_task_current_details)
 
-        with self.subTest("Restoring a 2.0 backup task with 2.1 manager"):
+        with self.subTest("Restoring an older version backup task with newer version manager"):
```

Comment:
If you can, if you have access to this information, please add the manager version to the `subTest` message.

Reply:
I thought about that, but considering that message is not printed anywhere, it's a useless effort. The message is strictly for clarity when reading the code, not for the output.